### DISCUSSION TITLE
How to enable password protection on an entry level

### DISCUSSION THREAD

**Discussion by sebszocinski** [statamic/cms#4738] [Q&A]
I'm trying to setup password protection per entry, ie an allowed password that only works for a single entry rather than site-wide.

Reading the docs here: https://statamic.dev/protecting-content#valid-passwords it mentions the following:

> A valid password is anything matching one of the passwords in the allowed list as configured on the page (or site-wide). This means that you can send three people three different passwords to access the same URL, each having their own way in.

To me that sounds like the `allowed` list of passwords can be set per entry, yet this doesn't seem to work.

I've tried:
```
protect:
  -
    type: password
    allowed: test321
```
and:
```
protect: password
allowed: test321
```
...Neither seem to work.
Am I missing something or are the docs unclear?

**Comment by edalzell** [comment]
the passwords go in the scheme: https://statamic.dev/protecting-content#configuring-schemes

**Reply by sebszocinski** [reply]
but there is no way of setting a unique password per entry?

**Reply by jasonvarga** [reply]
No. If the docs suggest this somewhere, it should be changed.

**Reply by sebszocinski** [reply]
https://statamic.dev/protecting-content#valid-passwords
> A valid password is anything matching one of the passwords in the allowed list as configured _**on the page**_ (or site-wide). 

To me `on the page` comes across as being able to set a password in the entry yaml / markdown?

**Reply by jasonvarga** [reply]
It should probably say "as configured on the scheme". Since you can configure a scheme per page, or site-wide.
